ec2/spark_ec2.py

```diff
@@ -931,6 +947,22 @@ def deploy_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
     shutil.rmtree(tmp_dir)
 
 
+# Deploy files in a given local directory to a cluster, WITHOUT parameter substitution.
+# Note that unlike deploy_files, this works for binary files.
+# Files are only deployed to the first master instance in the cluster.
+#
+# root_dir should be an absolute path to the directory with the files we want to deploy.
+def deploy_user_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
```

Comment:
`conn` isn't either. I just kept the same signature as deploy_files (since we may add substitution capability later). But  that's a pretty weak reason.
Want me to remove them @shivaram?